demo/values.tpl.yaml

```
@@ -1,18 +1,25 @@
 developer:
   urls:
-    diracx: http://{{ hostname }}:8000
+    diracx: http://{{ hostname }}:8000/api
+    diracx-web: http://{{ hostname }}:8000
```

I think we should use
```suggestion
    diracx: http://{{ hostname }}:8000
```
rather than implying we support any other layout for the paths.